AU2/database/AssassinsDatabase.py

```diff
@@ -44,12 +44,12 @@ def get(self, identifier: str) -> Assassin:
         """
         return self.assassins[identifier]
 
-    def get_identifiers(self) -> List[str]:
+    def get_identifiers(self, include_hidden: bool = False) -> List[str]:
         """
         Returns:
             list of identifiers sorted alphabetically
         """
-        return sorted([v for v in self.assassins], key=lambda n: n.lower())
+        return sorted([id for id, a in self.assassins.items() if include_hidden or (not a.hidden)], key=lambda n: n.lower())

```

Comment:
you don't need brackets around `not a.hidden`. also, `id` shadows the built-in local function of the same name - could you change this to `identifier` or similar?